common/src/com/thoughtworks/go/publishers/GoArtifactsManipulator.java

```diff
@@ -201,7 +201,7 @@ private String getDestPath(String file) {
     public void setProperty(JobIdentifier jobIdentifier, Property property) {
         try {
             String propertiesUrl = urlService.getPropertiesUrl(jobIdentifier, property.getKey());
```

Comment:
There's probably another issue here where we're building the url by concatenating the keys. Do we need URL escaping? (probably not, given that nobody has complained about it so far)